build.mjs

```diff
@@ -3,7 +3,20 @@ import fs from 'fs';
 import Jimp from 'jimp';
 import path from 'path';
 import readline from 'readline';
+import { parse } from 'envfile';
 import { fileURLToPath } from 'url';
+import { getPreviewBasePath } from './basePath.js';
+
+const envPresets = {
+  prod: {
+    REACT_APP_DEPLOY_ENV: 'PROD',
+    REACT_APP_PUBLIC_URL: 'https://push.org/',
+  },
+  preview: {
+    REACT_APP_DEPLOY_ENV: 'PREVIEW',
+    REACT_APP_PUBLIC_URL: `https://push-protocol.github.io${getPreviewBasePath()}`,
```

Comment:
`push-protocol.github.io`
can this part also come from env variables which we have in preview.yml?